README.md

```diff
@@ -1,36 +1,68 @@
-# Hello Microverse Project
+<a name="readme-top"></a>
 
-> This project has been built as part of the reviewers exercise or training.
-## Built With
+<div align="center">
 
-- HTML, CSS3
-- Github flow
-- VSCode
+  <h3><b>Microverse README Template</b></h3>
 
-## Authors
+</div>
 
-👤 **AJ193**
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+**[Hello Microverse]** is a simple html project containing one header.
+
+## 🛠 Built With <a name="built-with"></a>
+
+HTML and CSS
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] Good job :+1: but as the previous reviewer requested, Kindly use [Microverse Readme template](https://github.com/microverseinc/readme-template) and make sure all the main sections are present in yours. But now comparing your readme to this [Microverse Readme template](https://github.com/microverseinc/readme-template), you will see that you are missing some necessary headings like `key features`, `Future Features`, and ` Getting Started` that describe how a new developer could make use of your project by following some steps.